website/playground-versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "0.68.x",
```

Comment:
Oh interesting - how does this work when we have libraries in different versioning states?